common/util.go

```diff
@@ -22,8 +22,8 @@ import (
 
 // GetClientConfig return rest config, if path not specified, assume in cluster config
 func GetClientConfig(kubeconfig string) (*rest.Config, error) {
-	if kubeconfig != "" {
+	//if kubeconfig != "" {
```

Comment:
why is this commented?

Reply:
This came in as part of the operator logic and resulted in tests failing.

Reply:
which test failed?

Reply:
If I remember right it was a dependency issue with clientcmd, not at my machine so can't take a look

Reply:
Will have a look in the morning

Reply:
Cool. The output of this function depends on the environment you are trying to run the tests in. So I am inclined towards not testing this one function.

Reply:
Sounds good, Ill add an exclusion so we can uncomment it

Reply:
This has been resolved - I've updated Gopkg.lock to reflect the clientcmd dependency